registry_provider_version.go

```diff
@@ -34,15 +34,15 @@ type registryProviderVersions struct {
 
 // RegistryProviderVersion represents a registry provider version
 type RegistryProviderVersion struct {
-	ID                 string                      `jsonapi:"primary,registry-provider-versions"`
-	Version            string                      `jsonapi:"attr,version"`
-	CreatedAt          string                      `jsonapi:"attr,created-at,iso8601"`
-	UpdatedAt          string                      `jsonapi:"attr,updated-at,iso8601"`
-	KeyID              string                      `jsonapi:"attr,key-id"`
-	Protocols          []string                    `jsonapi:"attr,protocols,omitempty"`
-	Permissions        RegistryProviderPermissions `jsonapi:"attr,permissions"`
-	ShasumsUploaded    bool                        `jsonapi:"attr,shasums-uploaded"`
-	ShasumsSigUploaded bool                        `jsonapi:"attr,sasums-sig-uploaded"`
+	ID                 string                             `jsonapi:"primary,registry-provider-versions"`
+	Version            string                             `jsonapi:"attr,version"`
+	CreatedAt          string                             `jsonapi:"attr,created-at,iso8601"`
+	UpdatedAt          string                             `jsonapi:"attr,updated-at,iso8601"`
+	KeyID              string                             `jsonapi:"attr,key-id"`
+	Protocols          []string                           `jsonapi:"attr,protocols,omitempty"`
+	Permissions        RegistryProviderVersionPermissions `jsonapi:"attr,permissions"`
```

Comment:
Should be `*RegistryProviderVersionPermissions`

Reply:
Hm... this was changed per a comment from @brandonc because the permissions are required. Maybe I'm missing something? 🤔

Reply:
Right... This struct is never empty in the response so I didn't think a pointer is necessary for two bools